chord_metadata_service/restapi/utils.py

```diff
@@ -227,7 +227,7 @@ def monthly_generator(start: str, end: str) -> Tuple[int, int]:
     [end_year, end_month] = [int(k) for k in end.split("-")]
     last_month_nb = (end_year - start_year) * 12 + end_month
     for month_nb in range(start_month, last_month_nb):
```

Comment:
range is semi-open: `[start_month, last_month_nb)`: does this work as intended right now? because elsewhere where this is used seems to imply that it should be inclusive of end, but it hasn't been so far...

Reply:
Yes, nice catch. I already fixed an off-by-one error in here, but you're right that this misses the last month also.